EIPS/eip-5850.md

```diff
@@ -0,0 +1,71 @@
+---
+eip: 5850
+title: Complex Numbers stored in Bytes32 types
+description: Store real and imaginary parts of complex numbers in the lower 16 bytes and upper 16 bytes respectively of a bytes32 type.
+author: Paul Edge (@genkifs)
+discussions-to: https://ethereum-magicians.org/t/TBC
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-10-29
+---
+ 
+## Abstract
+
+This EIP proposes a natural way for complex numbers to be stored in the bytes32 data-type.
+
+## Motivation
```

Comment:
The motivation for this EIP is unclear/not obvious. Please add a strong motivation section.

Reply:
Motivation section extended.  I'm still not clear what is considered motivation and what is Rationale, but I'm happy to keep on improving the text.